src/WebJobs.Extensions.DurableTask/HttpApiHandler.cs

```diff
@@ -483,15 +482,10 @@ private static int GetIntQueryParameterValue(NameValueCollection queryStringName
                 DurableOrchestrationClientBase client = this.GetClient(request);
 
                 object input = null;
-                if (request.Content != null)
+                if (request.Content?.Headers?.ContentLength != 0)

```

Comment:
This is actually not fully correct. It should be this (I've made this change in another PR):
```suggestion
                if (request.Content != null && request.Content.Headers.ContentLength != 0)
```